tools/cppcheck/scripts/deploy_dashboard.py

```
@@ -0,0 +1,669 @@
+                    
+                    # Count C++17 opportunities
+                    cpp17_keywords = ['nullptr', 'override', 'auto', 'constexpr', 'noexcept']
+                    cpp17_issues = [i for i in data.get('issues', []) 
+                                   if any(k in i.get('message', '') for k in cpp17_keywords)]
+                    metrics['cpp17_opportunities'] = len(cpp17_issues)
+                    
+                    # Calculate coverage
+                    total_files = data.get('total_files', 1)
+                    if total_files > 0:
+                        metrics['code_coverage'] = (metrics['files_analyzed'] / total_files) * 100
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)